develop/dev-guide-sample-application-java-hibernate.md

```diff
@@ -1,374 +1,162 @@
-            System.out.printf("PlayerDAO.getPlayer:\n    => id: %s\n    => coins: %s\n    => goods: %s\n",
-                    testPlayer.getId(), testPlayer.getCoins(), testPlayer.getGoods());
-
-            // Count players amount.
-            Long count = (Long)playerDAO.runTransaction(session, playerDAO.countPlayers());
-            System.out.printf("PlayerDAO.countPlayers:\n    => %d total players\n", count);
-
-            // Print 3 players.
-            playerDAO.runTransaction(session, playerDAO.printPlayers(3));
-
-            // 4. Explore more.
-
-            // Player 1: id is "1", has only 100 coins.
-            // Player 2: id is "2", has 114514 coins, and 20 goods.
-            PlayerBean player1 = new PlayerBean("1", 100, 0);
-            PlayerBean player2 = new PlayerBean("2", 114514, 20);
-
-            // Create two players "by hand", using the INSERT statement on the backend.
-            int addedCount = (Integer)playerDAO.runTransaction(session,
-                    playerDAO.createPlayers(Arrays.asList(player1, player2)));
-            System.out.printf("PlayerDAO.createPlayers:\n    => %d total inserted players\n", addedCount);
-
-            // Player 1 wants to buy 10 goods from player 2.
-            // It will cost 500 coins, but player 1 cannot afford it.
-            System.out.println("\nPlayerDAO.buyGoods:\n    => this trade will fail");
-            Integer updatedCount = (Integer)playerDAO.runTransaction(session,
-                    playerDAO.buyGoods(player2.getId(), player1.getId(), 10, 500));
-            System.out.printf("PlayerDAO.buyGoods:\n    => %d total update players\n", updatedCount);
-
-            // So player 1 has to reduce the incoming quantity to two.
-            System.out.println("\nPlayerDAO.buyGoods:\n    => this trade will success");
-            updatedCount = (Integer)playerDAO.runTransaction(session,
-                    playerDAO.buyGoods(player2.getId(), player1.getId(), 2, 100));
-            System.out.printf("PlayerDAO.buyGoods:\n    => %d total update players\n", updatedCount);
-        } finally {
-            sessionFactory.close();
-        }
-    }
-}
-```
+    <Tip>
 
-## 第 3 步：运行代码
+    更多配置细节，可参考 [TiDB Dedicated 标准连接教程](https://docs.pingcap.com/tidbcloud/connect-via-standard-connection)。
 
-本节将逐步介绍代码的运行方法。
+    </Tip>
 
-### 第 3 步第 1 部分：TiDB Cloud 更改参数
+2. 运行以下命令，将 `env.sh.example` 复制并重命名为 `env.sh`：
 
-若你使用 TiDB Serverless 集群，更改 `hibernate.cfg.xml` 内关于 `hibernate.connection.url`、`hibernate.connection.username`、`hibernate.connection.password` 的参数：
+    ```bash
+    cp env.sh.example env.sh
+    ```
 
-```xml
-<?xml version='1.0' encoding='utf-8'?>
-<!DOCTYPE hibernate-configuration PUBLIC
-        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
-        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
-<hibernate-configuration>
-    <session-factory>
+3. 复制并粘贴对应的连接字符串至 `.env` 中。需更改部分示例结果如下。
```

Comment:
```suggestion
5. 复制并粘贴对应的连接字符串至 `env.sh` 中。需更改部分示例结果如下。
```

Reply:
@Icemap we are using `.env` or `env.sh`?

Reply:
That's a typo, it should be `env.sh`